docs/windows.md

```diff
@@ -65,28 +89,31 @@ the Windows Nodes in the demo.
 
 First install Antrea (v0.13.0+ is required for containerd).
 
+An example command that downloads and applies `antrea.yml` is like below:
```

Comment:
I think no need to make a new paragraph for this sentence. Just merge it to the last paragraph.

Reply:
You can say "The following command deploys Antrea v1.13.0" (or "... Antrea with the version specified by `<TAG>`") if you change `v1.13.0` to `<TAG>`).

Reply:
Updated.